RNTester/js/InputAccessoryViewExample.js

```diff
@@ -108,4 +104,14 @@ const styles = StyleSheet.create({
   },
 });
 
-module.exports = InputAccessoryViewExample;
+exports.title = '<InputAccessoryView>';
+exports.description =
+  'Example showing how to use an InputAccessoryView to build an iMessage-like sticky text input';
+exports.examples = [
+  {
+    title: 'Simple view with sticky input',
+    render: function(): React.Element<any> {
```

Comment:
We could probably just use `React.Node` here.

Reply:
Fixed here, should this be consistent elsewhere? I can follow up with another PR is so